src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg) {
```

Comment:
Add a constructor with an additional color parameter to set a custom color for the RightTriangle object 🎨